### PR TITLE
:book: Update test triggers on README

### PR DESCRIPTION
**Pull request by mquhuy**
Update triggers for release-0.8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ development. Here are the tests which run on different branches in BMO:
 **E2E tests:**
 
 - **/test metal3-bmo-e2e-test-pull** runs required BMO e2e tests on your PR.
-  Works for main and release-0.5 branches.
+  Works for main, release-0.5, release-0.6 and release-0.8 branches.
```
Comment (tuminoid) [inline README.md:46]: We can drop this line, as we don't have exceptions anymore.

File: README.md [hunk 2]
```diff
@@ -56,6 +56,15 @@ For more details on the e2e tests, see [test/e2e/README.md](test/e2e/README.md).
 - **/test metal3-ubuntu-e2e-integration-test-main** runs ansible integration
   tests with CAPM3 main branch and BMO **main** branch on Ubuntu
 
+**Release-0.8 branch:**
+
+- **/test metal3-centos-e2e-integration-test-release-1-8** runs CAPM3 e2e
+  integration tests with CAPM3 release-1.8 branch and BMO **release-0.8**
+  branch on Centos
+- **/test metal3-ubuntu-e2e-integration-test-release-1-7** runs ansible
```
Comment (tuminoid) [inline README.md:64]: Why 1.7?

**Review by tuminoid — COMMENTED** [review]
Also, fix the description, should read release-0.8 probably?

File: README.md (hunk 2 above)
Comment (mquhuy) [reply README.md]: Sorry, typo xD

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Review by kashifest — COMMENTED** [review]
/approve

**Pull request merged**